src/components/pages/UeberUns.vue

```diff
@@ -1,15 +1,106 @@
+                <h3>Transparenz</h3>
+                Wir sind sogar fast durchsichtig<br>
+                Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod
+                <h3>Ethereum</h3>
+                Etherum ist toll.<br>
+                Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod
+
+            </v-col>
+        </v-row>
+        <br><br>
+        <v-divider></v-divider>
+
+
+        <!--FAQ SECTION-->
+        <h1>FAQ</h1>
+        <br>
+        <v-expansion-panels
+                focusable
+                accordion
+                multiple
+        >
+            <v-expansion-panel class="qa">
+                <v-expansion-panel-header>
+                    <span class="question">
+                        <span class="font-weight-bold">Q:</span>
+                        Behaltet ihr Geld ein?
+                    </span>
+                </v-expansion-panel-header>
+                <v-divider></v-divider>
+                <v-expansion-panel-content class="answerBox">
+                    <v-row>
+                        <div>
+                            <span class="font-weight-bold">A:</span>
+                        </div>
+                        <v-col class="pa-0 pl-1">
+                            Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat.
+                        </v-col>
+                    </v-row>
+                </v-expansion-panel-content>
+            </v-expansion-panel>
+
+            <v-expansion-panel>
+                <v-expansion-panel-header>
+                    <span class="question">
+                        <span class="font-weight-bold">Q:</span>
+                        Ist das Frontend geil?
+                    </span>
+                </v-expansion-panel-header>
+                <v-divider></v-divider>
+                <v-expansion-panel-content class="answerBox">
+                    <v-row>
+                        <div>
+                            <span class="font-weight-bold">A:</span>
+                        </div>
+                        <v-col class="pa-0 pl-1">
+                         Die Antworten könnten die Bevölkerung verunsichern
+                        </v-col>
+                    </v-row>
+                </v-expansion-panel-content>
+            </v-expansion-panel>
+        </v-expansion-panels>
+    </v-card>
 </template>
 
 <script>
     export default {
-        name: "UeberUns"
+        name: "UeberUns",
     }
 </script>
 
 <style scoped>
+    .containerBox {
+        align-content: center;
+        line-break: auto;
+        margin-top: 15px;
+        margin-left:20px;
+        margin-right:20px;
+        padding:20px;
```

Comment:
Lerrzeichen zwischen : und wert fehlt
```suggestion
        margin-left: 20px;
        margin-right: 20px;
        padding: 20px;
```